includes/class-pubsubhubbub-publisher.php

```diff
@@ -117,7 +117,13 @@ public static function publish_to_hub( $feed_urls ) {
 	 */
 	public static function get_hubs() {
 		$endpoints = get_option( 'pubsubhubbub_endpoints' );
-		$hub_urls  = explode( PHP_EOL, $endpoints );
+
+		// Fail with an empty array when option retrieval fails
+		if ( false === $endpoints ) {
```

Comment:
I would add a more generic test, to also check for `null`!

```suggestion
		if ( ! $endpoints ) {
```